cwltest/__init__.py

```diff
@@ -127,14 +127,20 @@ def run_test(
                     test_number,
                     total_tests,
                     test.get("short_name"),
-                    test.get("doc"),
+                    test.get("doc", "").replace("\n", " "),
```

Comment:
Or `.strip()`? Not sure if there's a legit use of `\n` within the `"doc"` field?

Reply:
The use of `strip` only strips the last character.

This request is for the tests with multi-lined `doc` field such as [cwl-v1.2/conformance_tests.yaml#L3500](https://github.com/common-workflow-language/cwl-v1.2/blob/1d09972e5d92c9cc2c60781478e44ac10a0bef90/conformance_tests.yaml#L3500).

Of course it is possible to fix `conformance_tests.yaml` rather than `cwltest` but it needs much work because we already have three repositories (cwl-v1.0, cwl-v1.1, and cwl-v1.2) to be fixed.

Reply:
I added `strip` in addition to `replace` because in some cases the last character can be `" "`.